etl/etl.rb

```diff
@@ -0,0 +1,52 @@
+class ETL
+  def self.transform(old)
+    score=[]
```

Comment:
@neelgohel Space issues between `=`. You get now please follow the same for each and every program or exercise you write.